src/main/app/claims/claimStoreClient.ts

```diff
@@ -32,14 +44,12 @@ export class ClaimStoreClient {
     })
   }
 
-  saveClaim (draft: Draft<DraftClaim>, claimant: User): Promise<Claim> {
+  saveClaim (draft: Draft<DraftClaim>, claimant: User, admissionsAllowed: boolean): Promise<Claim> {
```

Comment:
Please make it more generic so that any feature can be requested this way e.g.

```
saveClaim (draft: Draft<DraftClaim>, claimant: User, optInFeature?: string): Promise<Claim>
```